src/views/PageReferenceWidget.vue

```diff
@@ -0,0 +1,117 @@
+<!--
+  - @copyright Copyright (c) 2022 Julien Veyssier <eneiluj@posteo.net>
+  -
+  - @author 2022 Julien Veyssier <eneiluj@posteo.net>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<template>
+	<div v-if="richObject" class="collective-page">
+		<div class="collective-page--image">
+			<span v-if="emoji"
+				class="page-emoji">
+				{{ emoji }}
+			</span>
+			<CollectivesIcon v-else
```

Comment:
If this is the page emoji, it probably should display the PageIcon for pages without emojis, not the CollectivesIcon, right?

Reply:
:+1: